Podfile.lock

```diff
@@ -62,7 +62,7 @@ PODS:
     - AppAuth/Core (~> 1.4)
     - GTMSessionFetcher/Core (~> 1.5)
   - GTMSessionFetcher/Core (1.6.1)
-  - Gutenberg (1.62.0):
+  - Gutenberg (1.61.1):
```

Comment:
WDYT about updating from develop @fluiddot ? Looks like the tag here is older than the one on develop.

Reply:
Sure, I'll update it, besides, there's a file conflict that should be resolved 👍 .

Reply:
I updated the branch with `develop` in [this commit](https://github.com/wordpress-mobile/WordPress-iOS/pull/17198/commits/3852874a673a6496cd48fde8b7c8ecc47276d30a).